VAMobile/src/components/PhotoUpload.tsx

```diff
@@ -0,0 +1,171 @@
+import { Image, Pressable, PressableProps } from 'react-native'
+import { ImagePickerResponse, launchCamera, launchImageLibrary } from 'react-native-image-picker'
+import { useTranslation } from 'react-i18next'
+import React, { FC, ReactNode, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { NAMESPACE } from 'constants/namespaces'
+import { VAIcon } from './index'
+import { themeFn } from 'utils/theme'
+import { useDestructiveActionSheet, useShowActionSheet } from 'utils/hooks'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import Box, { BoxProps } from './Box'
+import TextView from './TextView'
+import theme from 'styles/themes/standardTheme'
+
+type PhotoUploadProps = {
+  /** width of the photo */
+  width: number
+  /** height of the photo */
+  height: number
+  /** imagePickerResponse with asset to style for component and fileSize */
+}
+
+type StyledImageProps = {
+  /** prop to set image width */
+  width: number
+  /** prop to set image height */
+  height: number
+  /** Hardcoded radius of 50 due to circular design plan */
```

Comment:
If these props are hardcoded, do they still need to be props?

Reply:
This I'm strictly unsure of. I haven't done much tinkering with the StyledImage stuff and am just copying predecessors. I'm happy to change it if someone can test it out for me and make sure it's fine without this particular syntax.